devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/QuarkusTask.java

```diff
@@ -9,6 +9,7 @@
 
 import javax.inject.Inject;
 
+import org.apache.commons.lang3.StringUtils;
```

Comment:
I'm not sure we want to add a dependency to this one. I would drop it and have multiple conditions.

Reply:
I removed this dependency.